bindings/zeebe/command/cancel_instance_test.go

```diff
@@ -59,20 +69,12 @@ func TestCancelInstance(t *testing.T) {
 
 		req := &bindings.InvokeRequest{Data: data, Operation: cancelInstanceOperation}
 
-		mc := new(MockClient)
-		cmd1 := new(MockCancelInstanceCommandStep1)
-		cmd2 := new(MockCancelInstanceCommandStep2)
-
-		mc.On("NewCancelInstanceCommand").Return(cmd1)
-		cmd1.On("WorkflowInstanceKey", *payload.WorkflowInstanceKey).Return(cmd2)
-		cmd2.On("Send", mock.AnythingOfType("*context.emptyCtx")).Return(new(pb.CancelWorkflowInstanceResponse), nil)
+		mc := new(mockClient)
```

Comment:
nit: could just be `var mc mockClient`.

Reply:
If I do that, then I get the error message:

```
cannot use mc (type mockClient) as type zbc.Client in field value:
	mockClient does not implement zbc.Client (NewCancelInstanceCommand method has pointer receiver)
```

Reply:
Then you "share" the pointer to `mc` using `&`.